core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/LDAPConnectionContext.java

```diff
@@ -201,14 +201,14 @@ public DirContext getContext() throws UserStoreException {
         //if dcMap is not populated, it is not DNS case
         if (dcMap == null) {
             try {
-                context = getDirContext(environment);
+                context = getLdapContext(environment,null);
```

Comment:
Formatting issue. Lets have a space before the 2nd parameter. There are few other places as well with the same issue.